client/components/mobile-phone-verification/mobile-phone-verification.js

```diff
@@ -141,51 +186,6 @@ export default class MobilePhoneVerification extends React.Component {
     }
   }
 
-  hasPhoneTokenBeenSent() {
-    return sessionStorage.getItem(this.phoneTokenSentKey) !== null;
-  }
-
-  async createPhoneToken(resend=false) {
-    // do not send new SMS token if one has already been sent
-    if (!resend && this.hasPhoneTokenBeenSent()) {
-      return false;
-    }
-    const { orgSlug, mobile_phone_verification, language } = this.props;
-    const { errors } = this.state;
-    const { text } = mobile_phone_verification;
-    const self = this;
-    const url = createMobilePhoneTokenUrl(orgSlug);
-    return axios({
-      method: "post",
-      headers: {
-        "content-type": "application/x-www-form-urlencoded"
-      },
-      url
-    })
-      .then(() => {
-        // flag SMS as sent to avoid resending it
-        sessionStorage.setItem(self.phoneTokenSentKey, true);
-        toast.info(getText(text.token_sent, language));
-      })
-      .catch(error => {
-        const errorText = getErrorText(error);
-        logError(error, errorText);
-        toast.error(errorText);
-        this.setState({
-          errors: {
-            ...errors,
-            ...(errorText ? { nonField: errorText } : { nonField: "" }),
-          },
-        });
-      });
-  }
-
-  async handleLogout() {
-    const {orgSlug, logout, cookies} = this.props;
-    logout(cookies, orgSlug);
-    toast.success(logoutSuccess);
-  };
-
```

Comment:
why are you changing this?

Reply:
I see, it shouldn't happen on the latest master, if it does, let me know and I'll fix master (but the build is passing so it should not happen).

Reply:
ok